Snakefile

```diff
@@ -0,0 +1,63 @@
+
+
+rule all:
+    input:
+        required_outputs=
+
+
+rule remove_old_annotations:
+    """
+    Remove old annotations from Solr for a species
+    """
+    log:
+    input:
+    output:
+    shell:
+    """
+    """
+
+rule prep_working_dirs:
+    """
+    Prepare app and working directories
+    """
+    conda: "envs/cli.yaml"
+    log:
+    input:
+    output:
+    shell:
+    """
+    ./gradlew :cli:bootJar
+    mkdir ./bioentities ./bulk-analytics
+    """
+
+
+rule create_json_files:
+    """
+    Generate bioentity JSONL files for a species
+    """
+    conda: "envs/cli.yaml"
+    log:
+    input:
+    output:
+    shell:
+    """
+    java -jar ./cli/build/libs/atlas-cli-bulk.jar \
+    bioentities-json -o ./bioentities
```

Comment:
Here use instead the create_bioentities_jsonl.sh which directly uses the JAR from the conda package. This requires a ton of params (env vars) that need to be set, see the tests and the script itself.

Reply:
In general, I would suggest that a Snakemake for the workflow leaves elsewhere since it will require not only elements from this repo (index-bioentities) but also from index-gxa repo (for the analytics part). This repo should only contain the building blocks related to bioentities that that workflow needs to call.